tests/io/feed_parsers/ritzau_test.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013 - 2018 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+
+from superdesk.io.feed_parsers.ritzau import RitzauFeedParser
+from superdesk.tests import TestCase
+from superdesk.etree import etree
+import hashlib
+import os
+
+
+class BaseRitzauTestCase(TestCase):
+
+    vocab = [{'_id': 'genre', 'items': [{'name': 'Current'}]}]
+
+    def setUp(self):
+        with self.app.app_context():
+            self.app.data.insert('vocabularies', self.vocab)
+        dirname = os.path.dirname(os.path.realpath(__file__))
+        fixture = os.path.normpath(os.path.join(dirname, '../fixtures', self.filename))
+        provider = {'name': 'Test'}
+        with open(fixture, 'rb') as f:
+            self.root_elt = etree.fromstring(f.read())
+            self.item = RitzauFeedParser().parse(self.root_elt, provider)
+
+
+class RitzauTestCase(BaseRitzauTestCase):
+    filename = 'ritzau_news.xml'
+
+    def test_can_parse(self):
+        self.assertTrue(RitzauFeedParser().can_parse(self.root_elt))
+
+    def test_content(self):
+        item = self.item
+        self.assertEqual(item['version'], 1)
+        self.assertEqual(item['byline'], 'Via Ritzau')
+        self.assertEqual(item['headline'], 'PR / Spire Global, Inc.')
+        self.assertEqual(item['abstract'], 'UK Space Agency Backs Spire&apos;s Proposal to European Space Agency as'
+                                           ' Space Mission Provider')
+        self.assertEqual(hashlib.md5(item['body_html'].encode('utf-8')).hexdigest(), '8cda6746538782654a150c029b197fc0')
```

Comment:
imo it would be better to test it for something like `assertIn('<p>GLASGOW, Scotland', item['body_html'])`, I have no idea where this hash 8cda comes from

Reply:
it was to avoid copying huge part of text in the test. I can limit the test to just a part in clear text then